### DISCUSSION TITLE
invalid image reference

### DISCUSSION THREAD

**Discussion by ghost** [kyverno/policies#513] [Q&A]
Hello,

I'm receiving a strange warning regarding policy **block-stale-images/block-stale-images**. "policy block-stale-images/block-stale-images error: validation failure: failed to load context: invalid image reference %!s(<nil>), image reference must be a string"
But when I'm looking on the policy reporter the policy is pass. 
What can be the issue ?  The image is pulled on the statefulset, the pod are working fine ,and on the logs, the image is showed as a string.

**Comment by chipzoller** [comment]
Please provide details and complete reproduction steps.
